### PR TITLE
GH-420: Problem with using module name index.n4jsd in an N4JSD project when having dependency with another N4JSD project

### PR DESCRIPTION
**Pull request by qtran-n4**
Task: https://github.com/eclipse/n4js/issues/420

### CODE REVIEW

**Comment by qtran-n4** [comment]
@yoosiba Please review the PR. Thanks!

File: plugins/org.eclipse.n4js/src/org/eclipse/n4js/projectModel/ProjectUtils.xtend [hunk 1]
```diff
@@ -430,16 +430,15 @@ public class ProjectUtils {
...
+			val currentProject = n4jsCore.findProject(eObject.eResource.URI).orNull;
+			return targetProject == currentProject;
 		}
 
-		// for main modules we check containing project
-		val targetProject = n4jsCore.findProject(eoDescription.EObjectURI).orNull;
-		val currentProject = n4jsCore.findProject(eObject.eResource.URI).orNull;
-
-		return targetProject == currentProject;
+		return true;
```
Comment (yoosiba) [inline plugins/org.eclipse.n4js/src/org/eclipse/n4js/projectModel/ProjectUtils.xtend]: Shouldn't this be a false?

Comment (qtran-n4) [reply plugins/org.eclipse.n4js/src/org/eclipse/n4js/projectModel/ProjectUtils.xtend]: as discussed, it should be true :D

**Comment by qtran-n4** [comment]
Thanks! Build is green merging!

**Pull request merged**